cron/worker/main.go

```diff
@@ -61,20 +57,12 @@ func processRequest(ctx context.Context,
 		}
 		repoURLs = append(repoURLs, repoURL)
 	}
-	go func() {
```

Comment:
Are these requests going to be sequential? Can you please let me know the reason for this change?

Reply:
Yes, at least for now. With the parallel-thread approach something was off and the job was stuck indefinitely. I needed to unblock myself to further test this, so I changed this to sequential. Also adding a TODO here for this.

Reply:
OK, Thanks